src/steamship/agents/basic_chat/basic_chat_agent.py

```diff
@@ -0,0 +1,94 @@
+from typing import List
+
+from steamship import Block, SteamshipError
+from steamship.agents.basic_chat.basic_chat_output_parser import BasicChatOutputParser
+from steamship.agents.schema import Action, AgentContext, ChatAgent, ChatLLM
+from steamship.data.tags.tag_constants import RoleTag
+
+DEFAULT_PROMPT = """You are a helpful AI assistant.
+
+You chat with users are eager to engage on a variety of topics. You answer their questions, provide help thinking
+through challenges they have, and engage as a trusted assistant."""
+
+
+class BasicChatAgent(ChatAgent):
```

Comment:
some small concerns / things to ponder:

- is `ChatAgent` vs. `LLMAgent` (still) a useful distinction?  The lines seem incredibly blurred, and that may impact our agility here.
- is `Chat` overloaded?  the word, as used in our SDK has an backend api-specific meaning. if you want to use `Chat` to signal to our users that they can have a conversation with this LLM, then I think we need a different set of naming conventions, etc.